### ISSUE TITLE
Recover Teampass after crash server

### ISSUE THREAD

**Issue by nounours460** [nilsteampassnet/TeamPass#2291]
Hi,
We use Teampass since 1 year ago and we have a long electric crash.
The only files who we can save are /var/lib/mysql/*

We try to install a new fresh teampass and copy/cut the teampass database with the ibdata1 but when we lauch MySQl, he won't start. It say that iddata has changed.

More passwords are in teampass and it's a serious error for us. 

Can you help us please?

### Server configuration
**Operating system**: Debian Jessie

**Web server:** Apache 2 2.4.10

**Database:** Mysql 5.5.59

**PHP version:** 5.6.33

**Teampass version:** 2.1.27

**Teampass configuration file:** ??

**Comment by nilsteampassnet** [comment]
> More passwords are in teampass and it's a serious error for us.

Yes I can imagine.

The only thing I could advice you to check here is that you are using the exact same version of Mysql as in "crashed" server. Otherwise I think that there could be some inconsistencies ... perhaps the one you are facing.

Now if you don't have the saltkey file backuped somewhere, you will not be able to decrypt the passwords encrypted inside the DB.
Do you have this file?

**Comment by nounours460** [comment]
Yes I have saltkey file bachuped. But I'm not sure of version of MySQL because I test to reinstall it.

Have I another solution to recover my passwords?

**Comment by nilsteampassnet** [comment]
Good if you have the file.

Now regarding the mysql file, there is good pages on internet that explains how to use "root mysql files" and restore them.

Can you make us a feedback?

**Comment by nounours460** [comment]
Yes it's problem with mysql but I have now installed new version of teampass because it's impossible to recover my database. It's too hard to explain, I have had an answer on a mysql forum. But it's ok because lot of passwords can be recover with config file of applications or we know them.

Also I have new problem to define a password with complexity less than required. I enter a password with "low" complexity, is required "medium" but I authorised to create a first element with complexity less for the folder

And just a question on TeampassConnect (very good idea), in the login credentials, "URL" must say teampass is accessible on internet, it can't work with a local address?

Thanks a lot.
